pkgs/by-name/ti/tilt/binary.nix

```diff
@@ -33,4 +37,14 @@ buildGoModule rec {
     license = lib.licenses.asl20;
     maintainers = with lib.maintainers; [ anton-dessiatov ];
   };
-}
+}).overrideAttrs
+  (old: {
+    postInstall =
+      (old.postInstall or "")
+      + ''
```

Comment:
This should be conditional as mentioned above, and be included in the package definition above.